chain/chain/src/chain.rs

```diff
@@ -720,6 +721,13 @@ impl Chain {
         metrics::CHUNK_TAIL_HEIGHT.set(store.chunk_tail()? as i64);
         metrics::FORK_TAIL_HEIGHT.set(store.fork_tail()? as i64);
 
+        // TODO (#9989): Remove this config from chain
```

Comment:
This would be removed in the subsequent PR. Leaving it here for this one for correctness.